ExternalProcessLauncher/CS/ProcessLauncherSample/MainPage.xaml.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.Storage.Streams;
+using Windows.System;
+using Windows.UI.Core;
+using Windows.UI.Xaml.Controls;
+
+/* 
+* Notes:
+*   - Executables packaged as part of the AppX are allowed to run by default.
+      An example of such apps is SampleConsoleApp.exe which is included in the sample solution.
+*
+*   - System or any other app not packaged are also allowed to execute by adding it to the allowed list, AllowedExecutableFilesList, 
+*     using an absolute path; e.g. c:\mypath\myapp.exe
+*
+*     To add an exe to the allowed exe list, add to or replace the reg value in the reg command below:
+*
+*        reg.exe ADD "HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\EmbeddedMode\ProcessLauncher" /v AllowedExecutableFilesList /t REG_MULTI_SZ /d "c:\windows\system32\ipconfig.exe\0c:\windows\system32\tlist.exe\0"
+*
+*/
+namespace ProcessLauncherSample
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        public MainPage()
+        {
+            this.InitializeComponent();
+        }
+
+        private async Task RunProcess()
+        {
+            var options = new ProcessLauncherOptions();
+            var standardOutput = new InMemoryRandomAccessStream();
+            var standardError = new InMemoryRandomAccessStream();
+            options.StandardOutput = standardOutput;
+            options.StandardError = standardError;
+
```

Comment:
Extra blank line added from original.  Please remove.

Reply:
Removed the blank line.